### PR TITLE
Support for Telldus tellstick

### PR DESCRIPTION
**Pull request by swaner**
Added support for telldus tellstick: http://www.telldus.se/products/range. 
Reusing some of flossys code found in the forum: http://www.homegenie.it/forum/index.php?topic=310.0

Uses the open sourced telldus-core api (https://github.com/telldus/telldus/tree/master/telldus-core) which unfortunately needs to be installed separately in order to get it to work in homegenie.

Let me know if I need to do any changes.

### CODE REVIEW

**Comment by genielabs** [comment]
Hi @swaner thank you for this. It just need to be synched with latest revision and the PNG icon should have same size/style of other ones (30x28 px).

**Comment by swaner** [comment]
Hi @genielabs. Changed the icon and synchronized. Please review since I am not used to merging using GIT. If anything looks strange I can do a fresh checkout and redo all changes. Thanks again!

**Comment by swaner** [comment]
Nevermind @genielabs , I see that I have messed up _systemsettings.js. Will do a new fork instead and commit my changes there.

**Comment by swaner** [comment]
Please have a look now, I believe everything finally is in sync now.

File: BaseFiles/Common/Common.csproj [hunk 1]
```diff
@@ -4523,13 +4526,5 @@
     <Compile Include="dummy.cs" />
   </ItemGroup>
   <ItemGroup />
-  <ItemGroup>
```
Comment (genielabs) [inline BaseFiles/Common/Common.csproj]: why deleting these lines? these are required for new widgets editor.

File: BaseFiles/Common/Common.csproj [hunk 2]
```diff
@@ -4523,13 +4526,5 @@
     <Compile Include="dummy.cs" />
   </ItemGroup>
   <ItemGroup />
-  <ItemGroup>
-    <Folder Include="html\pages\control\widgets\homegenie\generic\locales\" />
-    <Folder Include="html\js\i18n\" />
-    <Folder Include="html\pages\configure\widgeteditor\" />
-    <Folder Include="html\js\codemirror-4.03\mode\css\" />
-    <Folder Include="html\js\codemirror-4.03\mode\htmlembedded\" />
```
Comment (genielabs) [inline BaseFiles/Common/Common.csproj]: why deleting these lines? these are required for new widgets editor.

Comment (genielabs) [inline BaseFiles/Common/Common.csproj]: many other files have the same issue, deleted lines sometimes inserted back.
I think it's best to checkout latest HomeGenie code, add your files and re-applying modifications to the html interface page.

**Comment by swaner** [comment]
Yes, I can see that. Strange, don't know the reason for this. Do you normally use visual studio for git operations or command line?
Anyway I have now  deleted my fork and created a new one. Checked out the latest HomeGenie code and applied my changes in one commit?
For some reason github thinks I wan't to add to my old fork. But maybe you can merge direcly from the new fork https://github.com/swaner/HomeGenie.

**Pull request closed without merging**